.github/workflows/release-pypi.yml

```diff
@@ -61,3 +61,6 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.pypi_password }}
+
+    # todo: upload checkpoint to legacy
```

Comment:
Great idea !

Reply:
yes, when it is done it shall be completely automatic, except adding version in pytest params